eng/Versions.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VersionPrefix>0.2.0</VersionPrefix>
+    <PreReleaseVersionLabel>prerelease</PreReleaseVersionLabel>
+    <RestoreSources>
+      $(RestoreSources);
+      https://api.nuget.org/v3/index.json;
+      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
```

Comment:
We're no longer publishing to myget. Please use:

`https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;`

Reply:
This came from copying it from Directory.Build.props. I'm not sure if we need it at all. I'll try removing it.